profile/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenTimelineIO
+[![OpenTimelineIO](images/OpenTimelineIO@3xDark.png#gh-light-mode-only)](http://opentimeline.io)
+[![OpenTimelineIO](images/OpenTimelineIO@3xLight.png#gh-dark-mode-only)](http://opentimeline.io)
```

Comment:
I see both logos at the same time in the rendered preview. Not sure if it's because of the preview or if the switch doesn't work

Reply:
I think they have to be on the same line for the effect to kick in.

Reply:
Or maybe they simply changed it to https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to. I wouldn't be surprised if they changed the way it's done since then.

Reply:
Note in the link provided:

> The old method of specifying images based on the theme, by using a fragment appended to the URL (#gh-dark-mode-only or #gh-light-mode-only), is deprecated and will be removed in favor of the new method described above.

After switching, the preview reflected my GH appearance setting.

Note the old version should still work, but I'm guessing the diff preview may be intentionally less "smart" for the markdown, whereas the `<picture>` might be kinda supported at a browser level.

Either way. I think it's looking good and we're on the GH paved path now!